src/components/UI/Controls/ConfirmationPrompt/index.tsx

```diff
@@ -126,6 +122,7 @@ const ConfirmationPrompt = React.forwardRef<
 ConfirmationPrompt.defaultProps = {
   confirmButton: 'Confirm',
   cancelButton: 'Cancel',
+  contentProps: { pad: 'medium', background: 'background-front' },
```

Comment:
Very minor thing: When this component is used, and we want to override the background color, we have to also set `pad` again. Also, if we wanted to add a new property, we would have to set `pad` and `background` again.

In other places we merge default props with the overrides at the component level
https://github.com/giantswarm/happa/blob/5912045903bbf35f01e33633cf6c79b10136fccb/src/components/UI/Inputs/RadioInput/index.tsx#L96

This makes sure that if we ever want to update the defaults, or add another default, we don't have to edit all files that use the component.